catalog-info.yaml

```diff
@@ -74,6 +75,9 @@ spec:
       repository: elastic/cloud-on-k8s
       provider_settings:
         trigger_mode: none
+        build_pull_requests: true
+        build_branches: true
+        build_tags: true
```

Comment:
Let's also make it possible to trigger the nightly pipeline via the API for anything.